### ISSUE TITLE
[Android] Not detecting that the plugin has already been linked

### ISSUE THREAD

**Issue by geof90** [rnpm/rnpm-plugin-link#27]
When I try to link the same plugin twice, it shows this message which seems good:
```
rnpm info Linking react-native-code-push android dependency 
rnpm info Android module react-native-code-push is already linked 
rnpm info Linking react-native-code-push ios dependency 
rnpm info iOS module react-native-code-push is already linked
```

However when i look at the settings.gradle file in Android, it looks like it is not detecting that it is already linked and it is inserting the same line twice
eg.
```
include ':app'
include ':react-native-code-push'
project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app')
include ':react-native-code-push' // <- this appears twice
project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app') // <- this appears twice
```

**Comment by geof90** [comment]
After more investigation, I see that this only happened to a project that was previously manually linked, not via RNPM.

So for example, this was my original gradle file
```gradle
include ':app', ':react-native-code-push' // note that i am including two projects in one line
project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app')
```
after running `rnpm link`, it becomes this invalid gradle file:

```gradle
include ':app'
include ':react-native-code-push'
project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app'), ':react-native-code-push'
project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app')
```

It looks like it is simply doing a string insertion right after `include ':app'`.

**Comment by grabbou** [comment]
cc: @Kureev probably related to the way you detect if app is installed? Maybe some regexp would be helpful here?

**Comment by Kureev** [comment]
well, it's because of `, ':react-native-code-push'` at the end of the line.

**Comment by lostintangent** [comment]
@Kureev Isn't is totally legitimate for the include statement to include a comma separated list of projects in addition to calling include N number of times? It would be awesome if the link command supported integration into existing projects that were using either syntax.

**Comment by geof90** [comment]
After looking at the code, I realize that the linkage detection code does not look at the gradle file at all, but looks at the MainActivity. I have a PR up that could potentially fix this for most users, please take a look.

**Comment by grabbou** [comment]
Thanks again for that. I also do think that checking for package definition is super error-prone given there are so many ways to construct them (e.g. the ones you've mentioned in your PR). Using build.gradle sounds much better.

**Comment by Kureev** [comment]
Fixed by merging in #29
